src/content/consul/install-landing.json

```diff
@@ -1,16 +1,62 @@
 {
-	"featuredTutorialsSlugs": [],
+	"featuredLearnContent": [
+		{
+			"collectionSlug": "consul/cloud-get-started"
+		},
+		{
+			"collectionSlug": "consul/gs-consul-service-mesh"
+		},
+		{
+			"collectionSlug": "consul/getting-started"
+		},
+		{
+			"tutorialSlug": "consul/kubernetes-api-gateway"
+		},
+		{
+			"tutorialSlug": "consul/consul-terraform-sync-intro"
+		},
+		{
+			"tutorialSlug": "consul/serverless-consul-with-lambda"
+		}
+	],
```

Comment:
@im2nguyen I see this is merged, but need to flag that `featuredLearnContent` is no longer supported so this will need to be updated in a follow-up PR before we ship. 😊